src/components/homepage/HomepageFooter/HomepageFooter.module.scss

```diff
@@ -0,0 +1,8 @@
+.communityBtn {
+    padding: 6px 16px;
```

Comment:
Need to update most of the values in this file.
- px to rem
- use predefined font-sizes